tests/test_transform.py

```diff
@@ -65,55 +68,3 @@ def test_transform(xslt_path: str, czi_xml_filename: str):
 
     # Test read from OME-Types
     from_xml(str(omexml))
-
-
-@pytest.mark.parametrize(
-    "czi_xml_filename, czi_subblock_xml_filename",
-    [
-        (
-            "s_3_t_1_c_3_z_5.czi.xml",
-            "s_3_t_1_c_3_z_5.czi.subblock.xml",
-        ),
-    ],
-)
-def test_transform_with_subblock(
```

Comment:
I was able to test support for subblocks in `test_transform()`, so this extra test is no longer needed.